charts/nexus3/values.yaml

```diff
@@ -76,6 +76,12 @@ service:
   #     name: docker-hosted
   #     containerPort: 8083
   #     host: nexus-docker-hosted.local
+  #   - port: 8084
+  #     name: docker-proxy
+  #     containerPort: 8084
+  #     hosts:
```

Comment:
I think if this is supported then `hosts` should be the recommended pattern and `host` should be documented as deprecated. So examples would want to be switched to `hosts`.

Reply:
I think it'd be best to update the old configuration to use `ports` as this isn't in the README.